CarrotMarket/CarrotMarket/Week03/ChatViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ChatViewController.swift
+//  CarrotMarket
+//
+//  Created by seozero on 4/26/25.
+//
+
+import UIKit
+
+import SnapKit
+
+final class ChatViewController: UIViewController {
+    private let chatList = ChatModel.dummy()
+    
+    private let tableView = UITableView(frame: .zero, style: .plain)
+    
+    override func viewDidLoad() {
+    
+        super.viewDidLoad()
+        
+        setLayout()
+        setDelegate()
+        register()
+        
+    }
+    
+    private func setLayout() {
+        self.view.backgroundColor = .white
+        self.view.addSubview(tableView)
+        tableView.snp.makeConstraints { $0.edges.equalToSuperview() }
+    }
+    
+    private func setDelegate() {
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    
+    private func register() {
+        tableView.register(ChatTableViewCell.self, forCellReuseIdentifier: ChatTableViewCell.identifier)
+    }
+}
+
+extension ChatViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 72
+    }
+}
+extension ChatViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 10
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: ChatTableViewCell.identifier, for: indexPath) as? ChatTableViewCell else { return UITableViewCell() }
```

Comment:
요렇게 길어지면 개행해주는 것도 가독성을 높이는 방법 !!! 
예를들면 이런식으루... 그치만 이건 개인 취향입니다 !

```suggestion
        guard let cell = tableView.dequeueReusableCell(
        withIdentifier: ChatTableViewCell.identifier, 
        for: indexPath
        ) as? ChatTableViewCell else { return UITableViewCell() }
```